### PR TITLE
feat(dropdown): add support for create item action

### PR DESCRIPTION
**Pull request by spirosikmd**
We add support for an action that can be used to create items. When a
search ends with no results, the "No results" message is hidden and only
the action is shown.

### Search with create item action

screen shot 2018-04-04 at 11 04 50

### Search with no results and create item action

screen shot 2018-04-04 at 09 44 01

### CODE REVIEW

**Comment by tzengerink** [comment]
I thought we were going for the option where the link to add the label is the first item in the list? Above the list of checkboxes so to say?

**Comment by spirosikmd** [comment]
@tzengerink The requirements changed. The footer should always stay at the bottom of the list.

**Comment by spirosikmd** [comment]
@kagnamazian the first image should now show that the first item is `highlighted` when the focus is on the search input.

**Comment by kagnamazian** [comment]
@spirosikmd I think, if the focus in the search-input. The Germany item selected should not be blue highlighted like that. But otherwise, I agree with the new Netherlands 'active' state. I don't know if it's just for the example or this behavior will exist with Javascript.
Preview of what I mean : 

screen shot 2018-04-04 at 11 00 52

**Comment by spirosikmd** [comment]
Yes, this is right @kagnamazian. A class is used in the docs that shouldn't be there and styles the item as active!

File: src/components/_dropdown.scss [hunk 1]
```diff
@@ -301,3 +309,102 @@
...
+//             <span class="sb-checkbox__faux">
+//               <svg class="sb-checkbox__checkmark" viewBox="0 0 8 7">
+//                 <path d="M7.68025,0 C5.067,1.97775 2.56075,4.6235 2.56075,4.6235 L0.7545,3 L0,3.7685 C0.74225,4.46625 2.46225,6.30925 3.044,7 C4.6815,4.26425 6.38375,2.224 8,0.3585 L7.68025,0 Z"></path>
+//               </svg>
+//             </span>
+//             <span class="sb-checkbox__text">Germany</span>
+//           </label>
+//         </div>
+//       </li>
+//       <li tabindex="-1" aria-label="Create er as new option">
```
[the remark this answers is not in the document]
Comment (spirosikmd) [reply src/components/_dropdown.scss]: I've tested it with the screen reader. It announces the button text once the button gets the focus.

**Pull request merged**